Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -611,7 +613,8 @@ enum ASPingEvent: String {
     case Share = "SHARE"
     case NewTab = "NEWTAB"
     case NewPrivateTab = "NEWPRIVATETAB"
-    case Bookmark = "BOOKMARK"
+    case AddBookmark = "ADDBOOKMARK"
+    case RemoveBookmark = "REMOVEBOOKMARK"
```

Comment:
Nit: On our desktop version of Activity Stream we use underscores between words to make it more clear. Could we do that here as well?